R/pkg/R/DataFrame.R

```diff
@@ -1290,8 +1290,10 @@ setClassUnion("characterOrColumn", c("character", "Column"))
 #' Sort a DataFrame by the specified column(s).
 #'
 #' @param x A DataFrame to be sorted.
-#' @param col Either a Column object or character vector indicating the field to sort on
+#' @param col A character or Column object vector indicating the fields to sort on
 #' @param ... Additional sorting fields
+#' @param decreasing A logical argument indicating sorting order for columns which are specified
+#'                   with their string names
```

Comment:
should we say something like `decreasing` is 'when a character vector is specified for col'?